src/test/scala/com.gu.contentapi.client/model/utils/CapiModelEnrichmentTest.scala

```diff
@@ -0,0 +1,161 @@
+package com.gu.contentapi.client.model.utils
+
+import com.gu.contentapi.client.model.v1.{Content, ContentFields, Tag}
+import com.gu.contentapi.client.utils.CapiModelEnrichment._
+import com.gu.contentapi.client.utils._
+import org.mockito.Mockito._
+import org.scalatest.{FlatSpec, Matchers}
+import org.scalatest.mockito.MockitoSugar
+
+
+class CapiModelEnrichmentTest extends FlatSpec with MockitoSugar with Matchers {
+
+  it should  "have a contentType of 'Media' when tag type/video is present" in {
```

Comment:
`have a designType`

Reply:
Damnit

Reply:
Sorry, that was uncalled for. Fixed now :(